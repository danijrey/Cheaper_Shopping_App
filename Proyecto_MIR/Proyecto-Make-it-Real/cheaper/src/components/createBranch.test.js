import React from 'react';
import {render} from '@testing-library/react';
import {MemoryRouter} from 'react-router-dom';
import CreateBranch from './createBranch';


describe ('CreateBranch',() =>{
    it ('should render correctly', ()=>{
    const {container} = render (
        <MemoryRouter> 
          <CreateBranch/>
        </MemoryRouter>
    );  
    expect (container.firstChild).toMatchSnapshot();
    });
});