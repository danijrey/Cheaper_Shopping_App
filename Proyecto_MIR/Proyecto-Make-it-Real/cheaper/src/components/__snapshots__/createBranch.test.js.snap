// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateBranch should render correctly 1`] = `
<span
  class="span-createBranch"
>
  <a
    class="btn-createBranch"
    data-testid="NewBranchLink"
    href="/providers/:id/branch/create"
  >
    Crear Sucursal
  </a>
</span>
`;
